main.js

```diff
@@ -32,3 +33,218 @@ const toggleMenu = () => {
+
+    const workDescription = document.createElement('p');
+    workDescription.classList.add('card-text');
+    workDescription.innerText = item.description;
+    workbody.appendChild(workDescription);
+    const projectTags = document.createElement('ul');
+    projectTags.classList.add('tag-section');
+    item.tags.forEach((tech) => {
+      const techItem = document.createElement('li');
+      techItem.classList.add('tag-div');
+      const tag = document.createElement('span');
+      tag.classList.add('tag');
+      tag.innerText = tech;
+      techItem.appendChild(tag);
+      projectTags.appendChild(techItem);
+    });
+
+    workbody.appendChild(projectTags);
+    const seeMoreBtn = document.createElement('button');
+    seeMoreBtn.type = 'button';
+    seeMoreBtn.classList.add('button');
+    seeMoreBtn.innerText = 'See Project';
+    seeMoreBtn.dataset.key = item.key;
+    seeMoreBtn.id = `myBtn${item.key}`;
+    workbody.appendChild(seeMoreBtn);
+    list.appendChild(workbody);
+    CardWrap.appendChild(list);
+  });
+};
+
+lunchCards();
+
+const modalTitle = document.querySelector('#myModal h2');
+const modalImg = document.querySelector('#myModal #pic1');
+const modalDesc = document.querySelector('#myModal .description');
+const modaltag1 = document.querySelector('#myModal .tag1');
+const modaltag2 = document.querySelector('#myModal .tag2');
+const modaltag3 = document.querySelector('#myModal .tag3');
+const modaltag4 = document.querySelector('#myModal .tag4');
+const modallive = document.querySelector('#myModal #live1');
+const modalgithub = document.querySelector('#myModal #github1');
+
+const modal = document.getElementById('myModal');
+const btn1 = document.getElementById('myBtn1');
+const btn2 = document.getElementById('myBtn2');
+const btn3 = document.getElementById('myBtn3');
+const btn4 = document.getElementById('myBtn4');
+
+const span = document.getElementsByClassName('close')[0];
+
+btn1.onclick = function () {
+  modal.style.display = 'block';
+  modalTitle.textContent = cards[0].title;
+  modalDesc.textContent = cards[0].description;
+  modalImg.src = cards[0].img;
+  modaltag1.textContent = 'HTML';
+  modaltag2.textContent = 'Ruby on Rails';
+  modaltag3.textContent = 'CSS';
+  modaltag4.textContent = 'JavaScript';
+  modallive.href = cards[0].liveDemoLink;
+  modalgithub.href = cards[0].githubLink;
+};
+
+btn2.onclick = function () {
+  modal.style.display = 'block';
+  modalTitle.textContent = cards[1].title;
+  modalDesc.textContent = cards[1].description;
+  modalImg.src = cards[1].img;
+  modaltag1.textContent = 'HTML';
+  modaltag2.textContent = 'Ruby on Rails';
+  modaltag3.textContent = 'CSS';
+  modaltag4.textContent = 'JavaScript';
+  modallive.href = cards[1].liveDemoLink;
+  modalgithub.href = cards[1].githubLink;
+};
+
+btn3.onclick = function () {
+  modal.style.display = 'block';
+  modalTitle.textContent = cards[2].title;
+  modalDesc.textContent = cards[2].description;
+  modalImg.src = cards[2].img;
```

Comment:
- Please on the popup for the desktop, please make the image and the text be aligned properly as in the Figma. on your design, the text description moves outward on the left making it not aligned properly with the image. Please make them align properly and set padding on the container of the popup, so it looks like that of the Figma design.

##### Your design
![Screenshot from 2022-06-15 12-09-22](https://user-images.githubusercontent.com/58771507/173813460-410d31a8-9731-4f3f-a6e6-06329d99979e.png)


##### Figma design
![Screenshot from 2022-06-15 12-08-14](https://user-images.githubusercontent.com/58771507/173813269-477a2833-62c1-4e5d-bb7d-6831a87021ad.png)